transformers/payable_resource_test.go

```diff
@@ -176,3 +178,20 @@ func TestUnitPayableResourceToPaymentDetails(t *testing.T) {
 		So(response.Items[0].ProductType, ShouldEqual, "late-filing-penalty")
 	})
 }
+
+func loadPenaltyDetails() *config.PenaltyDetailsMap {
```

Comment:
Do we want to grab this from somewhere (the yml file) or are we okay to keep this info local to this test class? Similar question for any test class that declares these details separately.

Reply:
I tried loading it from the yaml but it couldn't find the file. There is probably something I am doing wrong, but I could not figure it out. I have a unit test in for the yaml unmarshalling, so I am happy to keep this as just a test object in the class, but this is more a result of my lack of Go experience.